zzzz_modular_occulus/code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -11,15 +11,19 @@
 	embed = FALSE
 	sharp = FALSE
 
-/obj/item/projectile/bullet/pellet/shotgun/rubber/stinger	//used for the stinger grenade
+/obj/item/projectile/bullet/pellet/fragment/rubber/stinger	//used for the stinger grenade
+	name = "rubber pellet"
+	icon_state = "birdshot-1"
 	damage_types = list(BRUTE = 2)
-	agony = 18
-	pellets = 4
-	base_spread = 0 //causes it to be treated as a shrapnel explosion instead of cone
-	spread_step = 20
-	silenced = 1 //embedding messages are still produced so it's kind of weird when enabled.
-	no_attack_log = 1
-	muzzle_type = null
+	agony = 10
```

Comment:
This is a reduction from 18 (1800 total) agony to 12 agony (1000 from agony, 200 from brute.) Why are you reducing this?

Reply:
> 

Brute toned down to 1 per fragment.
Agony up to 17 agony per.